sdk/businesscreateuser/entity.go

```diff
@@ -0,0 +1,31 @@
+package businesscreateuser
+
+import (
+	"encoding/json"
+	"github.com/mazti/facebook-go-business-sdk/sdk"
+)
+
+const (
+	endpoint = "/business_users"
+)
+
+type Entity struct {
+	request *sdk.APIRequest
+
+	Id string `json:"id"`
+}
+
+func (ent *Entity) CreateBusinessUser(id string, context *sdk.APIContext) *APIRequestCreateBusinessUser {
```

Comment:
Have you take a look on package https://github.com/mazti/facebook-go-business-sdk/tree/master/sdk/businessuser
I think we can create a new function on that is good enough